index.html

```diff
@@ -3112,17 +3131,17 @@ <h3>Disputes</h3>
         </ul>
 
         <p>
-The mechanism for issuing a <code>DisputeCredential</code> is the same as
+The mechanism for issuing a disputed <a>credential</a> is the same as
```

Comment:
```suggestion
The mechanism for issuing a <code>DisputeCredential</code> is the same as
```

You don't issue a "disputed credential" ... rather, a credential that was issued in the past that you now want to dispute, so you issue a "DisputeCredential" to declare that you are disputing it.